### PR TITLE
Revision to PR #121, fixes issue #72 (missing tooltips)

### PR DESCRIPTION
**Pull request by cedricium**
Revised the issues with ESLint that I had with my other PR (#121).

Also, this PR only adds the titles to the toolbar buttons. Like @vladikoff had asked, I removed the additional key-bindings I had added in #121 and the titles are compatible with i18n.

### CODE REVIEW

File: locales/en-US/notes.properties [hunk 1]
```diff
@@ -16,3 +16,11 @@ disableSync=Disable Sync
...
 # {date}".
 syncComplete=Synced {date}
+
+# Titles for toolbar buttons
+fontSizeTitle=Font size
+boldTitle=Bold
+italicTitle=Italic
+strikethroughTitle=Strikethrough
+numberedListTitle=Numbered list
+bulletedListTitle=Bulleted list
```
Comment (vladikoff) [inline locales/en-US/notes.properties]: @cedricium could you please add a new line here (after `Bulleted list`)

File: src/sidebar/panel.js [hunk 2]
```diff
@@ -35,6 +35,21 @@ const quill = new Quill('#editor', {
...
 
+const size = document.getElementsByClassName('ql-size')[0],
+      bold = document.getElementsByClassName('ql-bold')[0],
+      italic = document.getElementsByClassName('ql-italic')[0],
+      strike = document.getElementsByClassName('ql-strike')[0],
+      ordered = document.getElementsByClassName('ql-list')[0],
+      bullet = document.getElementsByClassName('ql-list')[1];
+
+// Setting button titles in place of tooltips
+size.title = browser.i18n.getMessage('fontSizeTitle'),
```
Comment (vladikoff) [inline src/sidebar/panel.js]: @cedricium no need for `,` separated assignment here, just do
```js
size.title = thing
bold.title = otherThing
```

**Comment by cedricium** [comment]
@vladikoff want me to add the semi-colon after setting each title?

**Comment by vladikoff** [comment]
Hey @cedricium , a few things:


1. 

image

I don't see the tooltips when I hover on top of the icons. Did it work for you, how do you make them appear?

2. 

```
/home/travis/build/mozilla/notes/src/sidebar/panel.js
  46:54  error  Missing semicolon  semi
  47:50  error  Missing semicolon  semi
  48:54  error  Missing semicolon  semi
  49:61  error  Missing semicolon  semi
  50:61  error  Missing semicolon  semi
  51:60  error  Missing semicolon  semi
```

3. if you `npm run start-rtl` you will notice that for RTL languages we have an extra tooltip for text direction. We need to add a tooltip for that as well.

**Comment by vladikoff** [comment]
> @vladikoff want me to add the semi-colon after setting each title?

Yeap, thanks!

**Comment by cedricium** [comment]
Hey @vladikoff,

1. The "tooltips" are really just titles so when you hover over the buttons, the some text should appear like so:

image

It should just appear when you're hovering over the button however, not sure why it's not showing up.

2. Can easily add those semi-colons.
3. Will add the title for the RTL languages.

**Comment by cedricium** [comment]
@vladikoff the titles work for me, even with RTL running. Let me know if it still doesn't work for you.

File: locales/en-US/notes.properties [hunk 3]
```diff
@@ -16,3 +16,12 @@ disableSync=Disable Sync
 # structure doesn't work for your locale, you can translate this as "Last sync:
 # {date}".
 syncComplete=Synced {date}
+
+# Titles for toolbar buttons
```
Comment (flodolo) [inline locales/en-US/notes.properties]: I guess they're tooltips, not titles

Comment (vladikoff) [inline locales/en-US/notes.properties]: @cedricium can you update that and then we will merge! 😄

Comment (cedricium) [inline locales/en-US/notes.properties]: Changes made, thank you for letting me do this!  I find it very fun and rewarding being able to contribute. 🙌 

Would definitely like to tackle other issues if you don't mind.

Comment (Natim) [inline locales/en-US/notes.properties]: Yes please do :)

**Comment by vladikoff** [comment]
@cedricium Thanks so much! Great work on your first PR 👍

**Comment by pdehaan** [comment]
Congrats @cedricium, great job!

**Comment by cedricium** [comment]
Feels awesome! Thank you guys for all your help! Looking forward to contributing more 😄

**Comment by vladikoff** [comment]
@cedricium i saw you had a branch for https://github.com/mozilla/notes/issues/115 but didn't see a PR. Just reminding you so you don't forget to send that in.

**Comment by cedricium** [comment]
@vladikoff just need to work out a few kinks and test it then I'll make that PR :)

**Pull request merged**